packages/ionic/src/lib/__tests__/generate.ts

```diff
@@ -0,0 +1,66 @@
+import { prompt, getPages, tabsPrompt } from '../generate';
+
+describe('prompt', () => {
+
+  it('should return a file path', async () => {
+    const inquirer = {
+      prompt: jest.fn(() => {
```

Comment:
one reason one might do `jest.fn(fn)` is to use the jest mock utilities to check which calls occurred: https://facebook.github.io/jest/docs/mock-function-api.html